README.md

```diff
@@ -60,4 +60,7 @@
 1. Copy AirSim plugins to Unreal Project.
     - `cp /workspace/AirSim/Unreal/Plugins/ /path/to/Unreal/Project/` (e.g. '/workspace/UnrealProj/')
 1. `/workspace/UnrealEngine/Engine/Binaries/Linux/UE4Editor /path/to/Unreal/Project/<PROJECT_NAME>.uproject`
-1. Unreal Engine will prompt to rebuild AirSim. Click "Yes". First compile will take awhile.
\ No newline at end of file
+1. Unreal Engine will prompt to rebuild AirSim. Click "Yes". First compile will take awhile.
+
+## Others
+1. Remember to edit the blocks.sh script path for saving map.
```

Comment:
Which step of the Wenshan instructions does this correspond to?

Reply:
This corresponds to the mapping process using `sh blocks.sh`. When mapping is done, it saves to a path indicated in the blocks.sh which is hardcoded  to /home/wenshan.... thus directory not found.

Reply:
If we agree to shift tartanair workspace to mounted volume (see comments below), this is then too specific. Can omit from this repo, since this repo should encompass only building docker image, not every single details of running the various processes?

Reply:
Removed. Will have tartanair workspace in docker vol.